src/server/pachyderm_test.go

```diff
@@ -2339,8 +2339,12 @@ func TestGetLogs(t *testing.T) {
 	require.NoError(t, c.CreatePipeline(
 		pipelineName,
 		"",
-		[]string{"cp", path.Join("/pfs", dataRepo, "file"), "/pfs/out/file"},
-		nil,
+		[]string{"sh"},
+		[]string{
+			fmt.Sprintf("cp /pfs/%s/file /pfs/out/file", dataRepo),
+			"echo foo",
+			"echo foo",
```

Comment:
They're so that the lines below have some log messages to find. This might not be necessary if we add back in some of the default log statements.